Makefile

```diff
@@ -22,7 +23,7 @@ install-build-deps: ## Install dependencies (packages and tools)
 
 .PHONY: build
 build: generate-bindata-assets generate-kubernetes-types  ## Build eksctl
-	CGO_ENABLED=0 go build -tags "$(GO_BUILD_TAGS)" -ldflags "-X $(version_pkg).gitCommit=$(git_commit) -X $(version_pkg).builtAt=$(built_at)" ./cmd/eksctl
```

Comment:
What were the tags for?

Reply:
Ancient legacy.

See the commit message at https://github.com/weaveworks/eksctl/pull/944/commits/64f63b1424d4fb6ba4e23112466851e4d37d165b